tests/api_workflow/test_api_workflow_compute_worker.py

```diff
@@ -677,7 +677,7 @@ def test_get_compute_worker_runs(mocker: MockerFixture) -> None:
             last_modified_at=0,
         ),
     ]
-    assert mock_compute_worker_api.get_docker_runs.call_count == 2
+    mock_compute_worker_api.get_docker_runs.assert_called_once()
```

Comment:
The idea here is to test that this function is called with the correct arguments based on the call to `get_compute_worker_runs` so it's important to use `assert_called_once_with`.